templates/kvm/mk_cloud_image.erb

```diff
@@ -70,8 +72,10 @@ fi
 # solve bootstrapping of new kvm hosts where no vms exist already
 virsh pool-list | grep -qe "^\s*${pool_name}.*active" || {
     virsh pool-create-as --name "${pool_name}" --target "${images_dir}" --type dir --build
-    virsh pool-autostart "${pool_name}"
-    virsh pool-start "${pool_name}"
+    if dpkg --compare-versions ${VERSION_ID} lt 24.04 ; then
```

Comment:
This throws a shellcheck warning:
```
if dpkg --compare-versions ${VERSION_ID} ge 24.04 ; then
                           ^-----------^ SC2086 (info): Double quote to prevent globbing and word splitting.
```

Reply:
Added double quotes